internal/api/templates/client.tmpl

```diff
@@ -0,0 +1,29 @@
+// Start of User Generated Client
```

Comment:
Looks good, quick question, is this template needed or does the template with responses make it redundant?

Reply:
So, unfortunately this is a bit of a hack for now, what's going on is that the `client` target for `oapi-codegen` **_requires both template files_**, so basically I just separated the more "static" stuff that is boilerplate into the `client.tmpl` file and the more dynamic stuff that actually reflects on the schema in `client-with-responses.tmpl`.